src/components/spaces/spaces.njk

```diff
@@ -12,36 +12,95 @@
 {% block content %}
   {{ super() }}
 
-  {% if isAdmin or isManager or isBillingManager %}
-    <a href="{{ linkTo('admin.statement.dispatcher', {organizationGUID: organization.metadata.guid}) }}" class="govuk-link {% if routePartOf('admin.statement') %} active {% endif %}" class="govuk-link">
-      Billing
-    </a>
-  {% endif %}
+  <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-xl">Organisation</span>
+    {{ organization.entity.name }}
+  </h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+      <div class="paas-metric">
+        <h2 class="paas-metric__heading">
+          Quota usage
+        </h2>
+
+        <span class="paas-metric__figure">
+          {{
+            organization.entity.memory_allocated
+            | percentage(organization.entity.quota.entity.memory_limit)
+          }}
+        </span>
+      </div>
 
-  <div class="govuk-grid-row govuk-!-mb-r8 govuk-!-mt-r5">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">
-        This organisation has {{ spaces | length }} spaces
-      </h1>
+      <p class="paas-metric__context govuk-body">
+        Using
+        {{ organization.entity.memory_allocated | mbtogb }}
+        of
+        {{ organization.entity.quota.entity.memory_limit | mbtogb }}
+      </p>
     </div>
 
     <div class="govuk-grid-column-one-third">
-      <table class="govuk-table">
-        <tbody class="govuk-table__body">
-          <tr class="govuk-table__row">
-            <th class="govuk-table__header" scope="row">
-              Memory:
-            </th>
-            <td class="govuk-table__cell govuk-table__cell--numeric">
-              {{ (organization.entity.memory_allocated / 1024) | round(2) }}gb
-              of {{ (organization.entity.quota.entity.memory_limit / 1024) | round(2) }}gb
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <div class="paas-metric">
```

Comment:
Depends what you mean by metric related, I am definitely willing to change the name of this class, but I cannot think of a better name, ideas welcome. Right now it displays space "metrics" or space "info" (designs stolen from content data :) )

![Screen Shot 2019-09-27 at 07 59 04](https://user-images.githubusercontent.com/1482692/65748738-82a8ed00-e0fc-11e9-8187-dd6dc0227880.png)

The only thing that isn't a metric on this page is "Billable" (other state is "Trial")

Reply:
I think this is better named as `org-summary` or something to that effect then. I called this out because I know we've got more metric stuff to come in this codebase, and I didn't want us to end up having confusing CSS, with class names used in unexpected placed.

Reply:
Resolved in f7797bf156c1748a929069d2d0604ce25b7a29e7